sources/topology_optimization/topopt_brinkman_source.f90

```diff
@@ -200,7 +202,7 @@ end subroutine brinkman_source_term_free
   !! @param t The time value.
   !! @param tstep The current time-step.
   subroutine brinkman_source_term_compute(this, t, tstep)
-    class(brinkman_source_term_t), intent(inout) :: this
+    class(topopt_brinkman_source_term_t), intent(inout) :: this
     real(kind=rp), intent(in) :: t
     integer, intent(in) :: tstep
     type(field_t), pointer :: u, v, w, fu, fv, fw
```

Comment:
Using the registry here means you can't append this to the adjoint

Reply:
All i am doing here is a cleanup of the `use` statements